okhttp/src/jvmTest/java/okhttp3/CookieTest.kt

```diff
@@ -546,6 +550,69 @@ class CookieTest {
     assertThat(cookie.domain).isEqualTo("::1")
   }
 
+  @Test fun emptySameSite() {
+    assertThat(parse(url, "a=b; SameSite=")!!.sameSite).isEqualTo("")
+    assertThat(parse(url, "a=b; SameSite= ")!!.sameSite).isEqualTo("")
+    assertThat(parse(url, "a=b; SameSite=\r\t \n")!!.sameSite).isEqualTo("")
+  }
+
+  @Test fun spaceInSameSite() {
+    assertThat(parse(url, "a=b; SameSite=a b")!!.sameSite).isEqualTo("a b")
+  }
+
+  @Test fun trimLeadingAndTrailingWhitespaceFromSameSite() {
+    assertThat(parse(url, "a=b; SameSite= ")!!.sameSite).isEqualTo("")
+    assertThat(parse(url, "a= b; SameSite= Lax")!!.sameSite).isEqualTo("Lax")
+    assertThat(parse(url, "a=b ; SameSite=Lax ;")!!.sameSite).isEqualTo("Lax")
+    assertThat(parse(url, "a=\r\t \nb\n; \rSameSite=\n \tLax")!!.sameSite).isEqualTo("Lax")
+  }
+
+  @Test fun builderSameSiteTrimmed() {
```

Comment:
I want to call out that passing an empty string in is a valid operation. Should we require a non-empty string? This issue also applies to the `name` and `value` fields.

Reply:
No, I don’t think empty strings are special.